libs/net/node_network.py

```diff
@@ -0,0 +1,47 @@
+import ipaddress
+from typing import Final
+
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.node_network_state import NodeNetworkState
+
+from tests.network.libs.nodenetworkconfigurationpolicy import DEFAULT_OVN_EXTERNAL_BRIDGE
+from utilities.constants import IPV4_STR
+
+DEFAULT_ROUTE_V4: Final[str] = "0.0.0.0/0"
+
+
+class NodeInterfaceNotFoundError(Exception):
+    pass
+
+
+class NodeDefaultRouteNotFoundError(Exception):
+    pass
+
+
+def get_br_ex_network_info(node_name: str, client: DynamicClient) -> dict:
```

Comment:
Please use structures and not dicts. Is there a well defined structure that can represent this information without inventing one ourself?

Also, I have an issue with prefixing `get`, it is a nit and opinionated, but at a level I ask to change it to something like a `lookup`.